### ISSUE TITLE
Save\Close the changed file is too slow on macOS by exFAT format

### ISSUE THREAD

**Issue by Sincky** [typora/typora-issues#3496]
Version:0.9.9.32.1(4191)
Example:
step1: open an **existing** .md file with typroa (the Disk format is exFAT)
step2: change something
step3: Save(command+S) or Close the typora,it working very very slow :( 
![image](https://user-images.githubusercontent.com/30948192/81662064-a04e7b00-946f-11ea-845e-9f5148cdb814.png)

Also,i do these setps in APFS,it work very good.
I think Typora working in exFAT is not good.
(not try other systems)

**Comment by abnerlee** [comment]
What about other app like TextEditor.app?

**Comment by Sincky** [comment]
Sorry, I can't provide favorable information anymore, because I have changed the disk to APFS and everything is ok. 
I tried to reproduce this problem with a USB flash drive (exFAT), but failed to reproduce it. 
I remember when the problem occurs, it will prompt "The document ... is on a volume that does not support permanent version storage."I guess the problem is related to this prompt，but it not prompt in USB flash drive(exFAT).
A bit metaphysical.
